components/furnituremix/application/src/app/core/contract/StoreFrontConfiguration.ts

```diff
@@ -0,0 +1,13 @@
+import { Currency } from '~/lib/pricing/currencies';
+import { TenantConfiguration } from './TenantConfiguration';
+
+export type StoreFrontConfiguration = {
+    crystallize: {
+        tenantIdentifier: string;
+    };
+    language: string;
+    locale: string;
```

Comment:
These are singular. We probably want string[] to future proof? We should hold both all supported languages/locales and the active one (context).

Reply:
that would be the context here `StoreFrontConfiguration` is the configuration (in the context) of the store front